code/utilities/helpers/AzureBlobStorageHelper.py

```diff
@@ -9,20 +10,16 @@ def __init__(self, account_name: Optional[str] = None, account_key: Optional[str
         env_helper : EnvHelper = EnvHelper()
 
         self.account_name = account_name if account_name else env_helper.AZURE_BLOB_ACCOUNT_NAME
+        self.container_name: str = container_name if container_name else env_helper.AZURE_BLOB_CONTAINER_NAME
         self.account_key = account_key if account_key else env_helper.AZURE_BLOB_ACCOUNT_KEY
```

Comment:
@ross-p-smith we need to have support for both since there are customers who don't have access to add roles up until they have their production version ready.